test/vast/Conversion/Common/CoreToLLVM/binland-a.c

```diff
@@ -1,4 +1,4 @@
-// RUN: %vast-cc1 -vast-emit-mlir=hl %s -o - | %vast-opt-core-to-llvm | %file-check %s
+// RUN: %vast-cc1 -vast-emit-mlir=hl %s -o - | %vast-opt-irs-to-llvm | %file-check %s
```

Comment:
why irs-to-llvm when it is in `CoreToLLVM` folder?

Reply:
I will clean this when we update tests, really what that `%vast-opt-core-to-llvm` did was the same as the `irs-to-llvm` but just omitted some random passes in the middle.